hooks/queries/product/useGetProducts.ts

```diff
@@ -0,0 +1,29 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import { productAPI } from 'apis';
+
+const LIMIT = 10;
+
+const getProductsAPI = async ({ pageParam = 0 }) => {
+  const { data } = await productAPI.getProducts({ offset: pageParam });
+
+  return {
+    data,
+    current_page: pageParam,
+    isLast: data.length === 0 ? true : false,
```

Comment:
사소하지만 이렇게도 구현할 수 있습니다!
```suggestion
    isLast: data.length ? false : true,
```